Source/WebCore/html/HTMLScriptElement.cpp

```diff
@@ -113,16 +173,34 @@ String HTMLScriptElement::crossOrigin() const
     return parseCORSSettingsAttribute(attributeWithoutSynchronization(crossoriginAttr));
 }
 
-URL HTMLScriptElement::src() const
+String HTMLScriptElement::src() const
+{
+    return document().completeURL(sourceAttributeValue()).string();
+}
+
+ExceptionOr<void> HTMLScriptElement::setSrc(const std::variant<RefPtr<TrustedScriptURL>, String>& value)
 {
-    return document().completeURL(sourceAttributeValue());
+    ExceptionOr<String> stringValueHolder = WTF::switchOn(
+        value,
+        [this](const String& str) -> ExceptionOr<String> {
+            return trustedTypeCompliantString(TrustedType::TrustedScriptURL, *scriptExecutionContext(), str, "HTMLScriptElement src"_s);
+        },
+        [](const RefPtr<TrustedScriptURL>& trustedScriptURL) -> ExceptionOr<String> {
```

Comment:
Could just use a shorter name like `scriptURL`; no need to emphasize **trusted** in this local variable name, and names that are closer to just one word are typically easier to read. I think we could do the same with the local variables named `trustedScript`.